README.md

```
@@ -92,7 +121,7 @@ _Drop pipelines into any Python or TypeScript app with a few lines of code, no i
 
 4. You can run a pipeline from the canvas by pressing the ▶ button on the source node or from the `Connection Manager` directly.
 
-5. Deploy your pipelines on your own infrastructure.
+5. Deploy your pipelines - pick the path that fits:
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 2; Actual: 5; Style: 1/2/3

(MD029, ol-prefix)